src/index.html

```diff
@@ -49,6 +49,58 @@ <h2>Today's occupancy</h2>
 
         </form>
       </section>
+      <section class='guest-display hidden'>
```

Comment:
You have a great structure for the iterations, you went the same route I did using the select for room type. Just an FYI you can add a value to them, which you can get back the value to filter the rooms. You may be going a different route but its always an option.